src/main/java/com/jcabi/manifests/Manifests.java

```diff
@@ -187,6 +198,18 @@ public String get(final Object key) {
         return this.attributes.get(key);
     }
 
+    /**
+     * List all values for this key across all appended manifests in the order
+     * they were appended.
+     * @todo: #31 do not expose the internal list structure, or expose readonly
+     *  collection wrapper
+     * @param key The key of the manifest attribute
+     * @return The list of all found values
+     */
+    public List<String> getAll(final String key) {
```

Comment:
@pinaf Oh, seems like something qulice could be looking for. Have raised https://github.com/teamed/qulice/issues/769